app/main.py

```
@@ -2,24 +2,92 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: object, name: str) -> None:
+        self.private_name = "_" + name
+
+    def __get__(self, instance: type, owner: object) -> None:
+        return getattr(instance, self.private_name)
+
+    def __set__(self, instance: type, value: any) -> None:
+        if not isinstance(value, int):
+            raise TypeError()
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError()
+        return getattr(instance, self.private_name)
 
 
 class Visitor:
-   pass
+
+    def __init__(self,
+                 name: str,
+                 age: int,
+                 weight: float | int,
+                 height: float | int
+                 ) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(self,
+                 age: int,
+                 weight: float | int,
+                 height: float | int
+                 ) -> None:
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class ChildrenSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    def __init__(self,
+                 age: int,
+                 weight: float | int,
+                 height: float | int
+                 ) -> None:
+        if not (4 <= age <= 14):
+            raise ValueError("zły wiek")
+        elif not (80 <= height <= 120):
+            raise ValueError("zły wzrost")
+        elif not (20 <= weight <= 50):
+            raise ValueError("Zla waga")
+        else:
```

The logic for raising `ValueError` in `ChildrenSlideLimitationValidator` is incorrect. It should raise an error when the age, height, or weight is outside the specified range, not when it is within.